.github/workflows/release.yml

```
@@ -4,70 +4,745 @@ on:
+    outputs:
+      cargo_version: ${{ steps.meta.outputs.cargo_version }}
+      release_tag: ${{ steps.meta.outputs.release_tag }}
+      is_prerelease: ${{ steps.meta.outputs.is_prerelease }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v7
+
+      - name: Resolve tag and version
+        id: meta
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:17:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)